ui/userguide.html

```diff
@@ -168,244 +168,6 @@ <h3>Job notation</h3>
       <!-- End of row -->
       </div>
 
-      <div class="row">
-        <!-- Builds table -->
-        <div class="col-6">
-          <div class="card">
-            <div class="card-header">
-              <h3>Builds</h3>
-            </div>
-            <div class="card-body panel-spacing">
-              <table id="legend-builds" class="ug-btn-sm ug-btn-desc">
```

Comment:
I think the CSS definitions for `ug-btn-desc` and `ug-btn-sm` are now unused?

Reply:
Good one, I had looked at userguide.css but wrongly thought those tags were used in the notation table.